src/platform/plugins/shared/vis_types/timeseries/server/lib/vis_data/helpers/get_splits.ts

```diff
@@ -52,7 +53,11 @@ export async function getSplits<TRawResponse = unknown, TMeta extends BaseMeta =
     meta = get(resp, `aggregations.${series.id}.meta`) as TMeta | undefined;
   }
 
-  const color = new Color(series.color);
+  // FIXME: the series.color could be undefined even if it is typed as required. This happen due to
+  // a partially implemented Mock of Series and Panel in the process_bucket.test.ts
+  // The fallback follows the preexisting fallback used from the `color` library
+  // https://github.com/Qix-/color/blob/e188999dee229c902102ec37e398ff4d868616e5/index.js#L38-L41
+  const color = chroma.valid(series.color) ? chroma(series.color).hex() : '#000000';
```

Comment:
This color valid check needs to be applied everywhere when you the code use `new Color(c)` or simply `Color(c)`, if not you can end up throwing errors if the saved/retrieved/used color code is not valid